tests/test_lmdeploy/test_model.py

```diff
@@ -776,3 +776,32 @@ def test_phi3(model_path_and_name):
     ref = tokenizer.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
     res = model.messages2prompt(messages)
     assert res.startswith(ref)
+
+
+@pytest.mark.parametrize('model_path_or_name', [
+    'deepseek-ai/DeepSeek-R1-Distill-Llama-70B',
```

Comment:
May add all models in the group https://huggingface.co/collections/deepseek-ai/deepseek-r1-678e1e131c0169c0bc89728d to the test cases.